### PR TITLE
Add API accpeptance test for checking clear logfile

### PR DESCRIPTION
**Pull request by dpakach**
## Description

Add acceptance test for clear logfile using the testing API

## Checklist:

- [ ] Latest changes are published according to [instructions in README.md](https://github.com/owncloud/testing#publish-latest-version-as-github-release)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/TestingAppContext.php [hunk 1]
```diff
@@ -558,6 +558,56 @@ public function theAdministratorDeletesTheDirectoryUsingTheTestingApi($dir) {
...
+	}
+
+	/**
+	 * @Then log entries should decrease when the administrator clears the logfile
+	 *
+	 * @return void
+	 */
+	public function logEntriesShouldDecreaseWhenTheAdministratorClearsTheLogfile() {
+		$this->getLogfile();
+		$initialCount = \count($this->featureContext->getResponseXml()->data[0]);
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/TestingAppContext.php]: I guess we assume that the installation and startup will have put something in the log file, so ``initialCount`` will be enough greater than 0.

File: tests/acceptance/features/bootstrap/TestingAppContext.php [hunk 2]
```diff
@@ -558,6 +558,56 @@ public function theAdministratorDeletesTheDirectoryUsingTheTestingApi($dir) {
...
+		$this->clearLogFile();
+		$this->getLogfile();
+		$finalCount = \count($this->featureContext->getResponseXml()->data[0]);
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/TestingAppContext.php]: And we assume that the operation of clearing the log file does not itself add "a lot" of log entries.

Comment (dpakach) [reply tests/acceptance/features/bootstrap/TestingAppContext.php]: yes that is the case

**Comment by phil-davis** [comment]
@dpakach more conflicts for you :)

**Pull request merged**